index.js

```diff
@@ -1,2 +1,118 @@
 #!/usr/bin/env node
-require('./build/deploy')
+'use strict'
+
+const cloneDeep = require('lodash.cloneDeep')
+const co = require('co')
+const fs = require('mz/fs')
+const inquirer = require('inquirer')
+const path = require('path')
+const rsyncwrapper = require('rsyncwrapper')
+const yargs = require('yargs')
+
+function rsync(opts) {
+	return new Promise((yes, no) => {
+		rsyncwrapper(opts, (err, stdout, stderr, cmd) => {
+			const ret = { err, stdout, stderr, cmd }
+			return err ? no(ret) : yes(ret)
```

Comment:
Shouldn't it be `no(err)` instead of `no(ret)`, this will cause any try-catches to catch non-`Error`'s.

Might be better to do:

```js
const ret = { stdout, stderr, cmd }
if (err) {
    err.{ stdout, stderr, cmd } = { stdout, stderr, cmd }
    return no(err)
}
yes(ret)